docker/package/model.py

```diff
@@ -332,29 +333,22 @@ def gen_spec_file(self, build_deps, run_deps, out):
             f.write(file_contents)
 
     def gen_buildfile(self, out, binaries_dir=None):
+        name = self.name.replace("tezos", "octez")
```

Comment:
let's please change the `name` variable to something else.

This now makes really easy to get confused and wrong about which one is which